GoogleChromeLabs/ps-analysis-tool
# Rename “Frame details” section to Frame(s)

## Description

On the Cookies landing page, the `Frame details` section should be called `Frame` for singular value and `Frames` for more than one occurrence. This pull request contains the change for those two scenarios.

## Testing Instructions

1. Clone the repository and checkout the `fix/renaming-frame-detail-section` branch.
2. Run npm install to install all dependencies.
3. Start the server using the npm run build.
4. Open Chrome Devtools with the extension activated and check the cookies lading page
5. Check the Zero frame results should contain "0 Frames"
6. Check one frame scenario, it should contain "1 Frame"
7. Check more than one frame scenario, it should contain "x Frames"

## Screenshot/Screencast

**before:**
![Screenshot from 2024-01-02 22-09-52](https://github.com/GoogleChromeLabs/ps-analysis-tool/assets/330792/aa9ce366-b370-4a46-8a0f-678eaa419e56)

**after:**
![Screenshot 2024-01-03 at 09 45 21](https://github.com/GoogleChromeLabs/ps-analysis-tool/assets/330792/a043d20c-496a-4c58-8750-55704a044e8c)

---



Fixes #
